sockets.js

```diff
@@ -310,7 +348,7 @@ if (cluster.isMaster) {
 
 		case '!': // !socketid
 			// destroy
-			socketid = data.substr(1);
+			socketid = +data.substr(1);
```

Comment:
??? why though

Reply:
I thought numbers took up less space on the heap than strings for some reason. Double checking though I'm pretty sure I was wrong about ti